### ISSUE TITLE
enhance jwerty library to provide api

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#981]
similar to the voicecommander enhancements coming we should be able to attach a data property to anything (like data-jwerty-key) and have it fire a click event against it. This could be the built in behavior as its the most common. Anyone wanting to do other things could obviously just implement jwerty syntax.

It also would be good to have a built in way of modifying the key combo to press via UI. Right now it's shift+alt as that avoids most in system settings that could interfere.

**Comment by btopro** [comment]
with this for the selector, we just need to know the key pressed (so like letter k or whatever) then we can look for the `$("[data-jwerty-key='" + key + "']").click();` event. to trigger it. We've got it currently discovering everything pretty quickly but need it to do the binding.
